Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM rust
+
+RUN apt-get update && \
+    apt-get install -y cmake && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY . /opt/tarpaulin/
+
+RUN cd /opt/tarpaulin/ && \
+    cargo install && \
+    rm -rf /opt/tarpaulin/ && \
+    rm -rf /usr/local/cargo/registry/
```

Comment:
These are to make the image a bit smaller. And the registry would be out of date for users of this image anyway, between when we build it and they use it.